### ISSUE TITLE
Training LapIRN with Dice loss

### ISSUE THREAD

**Issue by rohitrango** [cwmok/LapIRN#32]
Hi,

Thank you for making the code public. I'm trying to augment LapIRN to incorporate anatomical information as well.
However, adding this loss seems to break the training process and the dice loss quickly goes up to 1. 

Have you tried incorporating dice loss into any of your networks?

**Comment by cwmok** [comment]
Hi @rohitrango,
I did! and it works well with the data from the L2R challenge.

Here is the dice loss I tried:
nnUnet softdice: https://github.com/MIC-DKFZ/nnUNet/blob/nnunetv1/nnunet/training/loss_functions/dice_loss.py

You can also try a lower learning rate, e.g., 1e-5.

**Comment by rohitrango** [comment]
do you have the code for this?

**Comment by cwmok** [comment]
loss_dice = SoftDiceLoss()
                
Y_label = Y_label.cuda().float()
X_Y_label = transform(X_label, F_X_Y.permute(0, 2, 3, 4, 1), grid_unit_full)
dice_loss = loss_dice(X_Y_label, Y_label)

Something like this.

**Comment by rohitrango** [comment]
Okay. I have a similar training loss script as well. Maybe I'll try reducing the learning rate or play around with the hyperparameters. 

Thanks for the help.

**Comment by rohitrango** [comment]
Okay. It looks like an issue with the Pytorch version used to train the model (I downgraded to Pytorch1.9 and it works). This seems to be a common occurance in the codebases, but no one seems to know exactly what has changed between these versions.
